src/app/tests/suites/certification/Test_TC_MC_3_8.yaml

```diff
@@ -45,6 +45,8 @@ tests:
 
     - label: "Reads the Status attribute"
       PICS: PICS_SKIP_SAMPLE_APP
+      # This is nonsense: Application Launcher and Application Basic are never
```

Comment:
actually, they can be (but with different feature codes). The application launcher on a content app endpoint can be used to start/stop the app. Don't get me started on the debate which resulted in this confusing outcome.